plugins/src/webrtcsink/imp.rs

```diff
@@ -454,18 +475,46 @@ fn setup_encoding(
     Ok((enc, conv_filter, pay))
 }
 
-fn lookup_remote_inbound_rtp_stats(stats: &gst::StructureRef) -> Option<gst::Structure> {
+fn lookup_remote_inbound_loss_round_trip_time(stats: &gst::StructureRef) -> (f64, f64) {
+    let mut inbound_rtp_stats: Vec<gst::Structure> = Default::default();
+
     for (_, field_value) in stats {
         if let Ok(s) = field_value.get::<gst::Structure>() {
             if let Ok(type_) = s.get::<gst_webrtc::WebRTCStatsType>("type") {
                 if type_ == gst_webrtc::WebRTCStatsType::RemoteInboundRtp {
-                    return Some(s);
+                    inbound_rtp_stats.push(s);
                 }
             }
         }
     }
 
-    None
+    let mut rtt = 0.;
+    let mut sum_fraction_lost = 0.;
+    let mut n_rtts = 0u64;
+    let mut n_fraction_losts = 0.;
+    let _ = inbound_rtp_stats
+        .iter()
+        .map(|s| {
+            s.get::<f64>("round-trip-time")
+                .map(|time| {
+                    rtt += time;
+                    n_rtts += 1;
+                })
+                .ok();
+
+            s.get::<f64>("fraction-lost")
+                .map(|f_lost| {
+                    sum_fraction_lost += f_lost;
+                    n_fraction_losts += 1.;
+                })
+                .ok();
+        })
+        .collect::<()>();
+
+    let loss_percentage = (sum_fraction_lost / n_fraction_losts) * 100.;
```

Comment:
This is related to what we discussed, this is not correct, you can't just make an average of loss percentages to calculate the overall loss percentage :)

Reply:
I removed the weight, but I see you were just ignoring it all for the RTT and were using only the first value, so this is more correct in any case.

Reply:
Not sure it matters whether things were incorrect before, what we want here is to operate on the absolute numbers, and track those from one report to the next ;)

Reply:
The stat is just not present here, using the percentage is correct here.